bin/onlyswaps-verifier/src/verifier.rs

```diff
@@ -0,0 +1,173 @@
+use crate::chain_state::NetworkBus;
+use crate::chain_state_pending::{RequestId, Verification};
+use crate::chain_state_resolver::ChainStateResolver;
+use crate::cli::StartArgs;
+use crate::config::{AppConfig, ConfigFile};
+use crate::evaluator::Evaluator;
+use crate::retry_runtime::RetryScheduler;
+use crate::signing::{NetworkedSigner, OnlySwapsSigner, VerifiedSwap};
+use crate::transport::create_libp2p_transport;
+use crate::verification_bus::VerificationBus;
+use agent_utils::healthcheck_server::HealthcheckServer;
+use agent_utils::monitoring::init_monitoring;
+use config::file::load_mapped_config_file;
+use std::sync::Arc;
+use tokio_stream::StreamExt;
+
+pub async fn start_verifier(args: StartArgs) -> anyhow::Result<()> {
+    let app_config = load_mapped_config_file::<ConfigFile, AppConfig>(args.config_path)?;
+    let healthcheck_server = HealthcheckServer::new(
+        app_config.agent.healthcheck_listen_addr,
+        app_config.agent.healthcheck_port,
+    )
+    .await?;
+    init_monitoring(&app_config.agent)?;
+
+    // listen for OS signals or any of the tasks closing and shut down either gracefully
+    // or noisily with errors
+    let mut sigterm = tokio::signal::unix::signal(tokio::signal::unix::SignalKind::terminate())?;
+    let mut sigint = tokio::signal::unix::signal(tokio::signal::unix::SignalKind::interrupt())?;
+    tokio::select! {
+        _ = sigterm.recv() =>
+            Ok(()),
+
+        _ = sigint.recv() =>
+            Ok(()),
+
+        _ = tokio::signal::ctrl_c() =>
+            Ok(()),
```

Comment:
I still find that formatting terrible
```suggestion
        // signal for graceful shutdown
        _ = sigterm.recv() => Ok(()),
        _ = sigint.recv() => Ok(()),
        _ = tokio::signal::ctrl_c() => Ok(()),
```